elide-integration-tests/src/test/java/com/yahoo/elide/async/integration/tests/TableExportIT.java

```diff
@@ -0,0 +1,1441 @@
+                                        type("tableExport"),
+                                        id("edc4a871-dff2-4054-804e-d80075cf830a"),
+                                        attributes(
+                                                attr("query", "/book?sort=genre&fields%5Bbook%5D=title"),
+                                                attr("queryType", "JSONAPI_V1_0"),
+                                                attr("status", "QUEUED"),
+                                                attr("asyncAfterSeconds", "0"),
+                                                attr("resultType", "CSV")
+                                        )
+                                )
+                        ).toJSON())
+                .when()
+                .post("/tableExport")
+                .then()
+                .statusCode(org.apache.http.HttpStatus.SC_CREATED)
+                .body("data.id", equalTo("edc4a871-dff2-4054-804e-d80075cf830a"))
+                .body("data.type", equalTo("tableExport"))
+                .body("data.attributes.status", equalTo("PROCESSING"))
+                .body("data.attributes.result.recordCount", nullValue())
+                .body("data.attributes.result.responseBody", nullValue())
+                .body("data.attributes.result.httpStatus", nullValue());
+
+
+        int i = 0;
+        while (i < 1000) {
+            Thread.sleep(10);
+            Response response = given()
+                    .accept("application/vnd.api+json")
+                    .get("/tableExport/edc4a871-dff2-4054-804e-d80075cf830a");
+
+            String outputResponse = response.jsonPath().getString("data.attributes.status");
+
+            // If Table Export is created and completed
+            if (outputResponse.equals("COMPLETE")) {
+
+                // Validate TableExport  Response
+                response
+                        .then()
+                        .statusCode(HttpStatus.SC_OK)
+                        .body("data.id", equalTo("edc4a871-dff2-4054-804e-d80075cf830a"))
+                        .body("data.type", equalTo("tableExport"))
+                        .body("data.attributes.queryType", equalTo("JSONAPI_V1_0"))
+                        .body("data.attributes.status", equalTo("COMPLETE"))
+                        .body("data.attributes.result.message", nullValue())
+                        .body("data.attributes.result.recordCount", equalTo(3))
+                        .body("data.attributes.result.url",
+                                equalTo("http://localhost:" + port + "/export/edc4a871-dff2-4054-804e-d80075cf830a"))
+                        .body("data.attributes.result.httpStatus", equalTo(200));
+
+                assertEquals("\"title\"\n"
+                        + "\"For Whom the Bell Tolls\"\n"
+                        + "\"Song of Ice and Fire\"\n"
+                        + "\"Ender's Game\"\n", getStoredFileContents("edc4a871-dff2-4054-804e-d80075cf830a"));
```

Comment:
Shouldn't this be downloading the file via the API and using that for comparison?